production/ksonnet/README.md

```diff
@@ -14,9 +14,12 @@ In your config repo, if you don't yet have the directory structure of Tanka set
 
 ```bash
 # create a directory (any name works)
-$ mkdir config && cd config/
+$ mkdir promtail && cd promtail/
 $ tk init
-$ tk env add loki --namespace=loki
+$ tk env add promtail --namespace=loki
```

Comment:
This doc deploy both, `promtail` and `loki`, which it should probably stay `loki` here